boards/arm64/renesas_rcar/gen3/common_salvator_xs_h3ulcb.dts

```diff
@@ -1,18 +1,12 @@
 /*
- * Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+ * Copyright (c) 2023 EPAM Systems
```

Comment:
Ugh. This is gross. Never ever remove  other's copyright notes.

Also newer put our copyright notes on a copied code  (of course, if original code was written by not us).

You can add a second copyright note if you did major changes to the file, though